registration/include/pcl/registration/impl/ia_ransac.hpp

```diff
@@ -97,7 +97,7 @@ SampleConsensusInitialAlignment<PointSource, PointTarget, FeatureT>::selectSampl
   sample_indices.clear();
   while (static_cast<int>(sample_indices.size()) < nr_samples) {
     // Choose a sample at random
-    int sample_index = getRandomIndex(static_cast<int>(cloud.size()));
+    const auto sample_index = getRandomIndex(static_cast<int>(cloud.size()));
```

Comment:
Do we need the static casts to `int`? Or the `int` type of `nr_samples`

Reply:
I think I did what you asked. Please check if that is what you meant